### ISSUE TITLE
Shadow() seems not to be working with data supplied

### ISSUE THREAD

**Issue by ckoenig2** [choi-phd/TestDesign#118]
### Package Version
1.3.3
### Current Behavior
`Shadow()` ignores (?) `data` argument and throws the following error: `either 'data' or 'true_theta' must be supplied`.
`Shadow()` works when `true_theta` is supplied.
### Expected Behavior
`Shadow()` should work when only `data` is supplied.
### Steps to Reproduce
```
library(TestDesign)
true_theta = rnorm(2) 
resp_data = simResp(itempool_reading, true_theta)
colnames(resp_data) = itempool_reading_data$ID
cfg1 = createShadowTestConfig(refresh_policy = list( method = "STIMULUS" ), MIP = list(solver = 'RSYMPHONY'))
solution = Shadow(cfg1, constraints_reading, data = resp_data)
```

**Comment by SangdonLim** [comment]
This issue is now fixed by #120 and will be released in the next version.

**Comment by SangdonLim** [comment]
TestDesign 1.3.4 is released today (7/22/2022) and incorporates a fix to this issue.
